configure: do not hard-code host paths

configure hard-codes include dirs to contain /usr/include/ncurses.
Needless to say this does not work well for cross-compilation.

Remove that directory from the include search dirs.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
Cc: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
[bernd.kuhls@t-online.de: update for 2016-07-R1]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -durN kismet-Kismet-2014-02-R1.orig/configure.in kismet-Kismet-2014-02-R1/configure.in
--- kismet-Kismet-2014-02-R1.orig/configure.ac	2014-02-19 05:37:43.000000000 +0100
+++ kismet-Kismet-2014-02-R1/configure.ac	2014-12-26 16:46:55.770692349 +0100
@@ -406,9 +406,6 @@
 )
 AC_SUBST(wantclient)
 
-# Add additional cflags since some distros bury panel.h
-CPPFLAGS="$CPPFLAGS -I/usr/include/ncurses"
-
 termcontrol="none";
 
 if test "$wantclient" = "yes"; then
